neural_compressor/torch/quantization/config.py

```diff
@@ -145,6 +153,11 @@ def get_model_info(model: torch.nn.Module) -> List[Tuple[str, Callable]]:
         logger.debug(f"Get model info: {filter_result}")
         return filter_result
 
+    @classmethod
+    def get_config_set_for_tuning(cls) -> Union[None, "RTNConfig", List["RTNConfig"]]:
+        # TODO fwk owner needs to update it.
+        return RTNConfig(weight_bits=[4, 6])
```

Comment:
Hi @xin3he @violetch24,  we should update the argument values before announcing the auto-tune API.